vcfa/resource_vcfa_content_library.go

```diff
@@ -219,15 +218,15 @@ func resourceVcfaContentLibraryImport(_ context.Context, d *schema.ResourceData,
 	tmClient := meta.(ClientContainer).tmClient
 
 	idSplit := strings.Split(d.Id(), ImportSeparator)
-	if len(idSplit) > 2 {
-		return nil, fmt.Errorf("invalid import identifier '%s', should be either <%s name>, or <%s name>%s<%s name>", labelVcfaContentLibrary, labelVcfaOrg, labelVcfaContentLibrary, d.Id(), ImportSeparator)
+	if len(idSplit) != 2 {
+		return nil, fmt.Errorf("invalid import identifier '%s', should be <%s name>%s<%s name>", d.Id(), labelVcfaOrg, ImportSeparator, labelVcfaContentLibrary)
```

Comment:
Shall we just make a detail that for provider CL, the Org must be "System". I saw it in the docs elsewhere, but maybe would be useful to emit it here as well.

Reply:
Done